tests/containers/validate_btrfs.pm

```diff
@@ -57,16 +57,16 @@ sub _test_btrfs_thin_partitioning {
 
 sub _test_btrfs_device_mgmt {
     my ($rt, $dev_path) = @_;
-    my $tw         = 'registry.suse.de/home/favogt/jumpmeta/images_tw/opensuse/tumbleweed:latest';
+    my $container  = 'registry.opensuse.org//suse/containers/caps/2/containers/registry/harbor-db:2.0.2';
```

Comment:
This variable looks like magic to me :slightly_smiling_face: 

Perhaps it would make sense to move this to a test SETTING instead. Are we sure that this container image will not change over time? Especially the version number at the end makes me think this might be better to make a variable out of it. What do you think?

Reply:
the amazing thing is that the image is not anymore in  registry.opensuse.org. and yes i think also that it is something magic. But i wouldnt make it a variable in the settings. rather pick another. what do you say about cloud/platform/stack/rootfs/images/sle15:latest

Reply:
If you are convinced that the image is reliable enough, then we can hard-code it here.
I'm just thinking that changing a SETTING in osd is much cheaper than changing a hard-coded link here (e.g. verification runs ecc.) But it's also something that we can change anytime in the future if it becomes a problem.

In short: :+1: